src/main/java/pl/craftserve/radiation/RadiationPlugin.java

```diff
@@ -97,14 +97,6 @@ public void onEnable() {
         Logger logger = this.getLogger();
         this.saveDefaultConfig();
 
-        try {
-            this.radiationNmsBridge = this.initializeNmsBridge();
-        } catch (Exception e) {
-            logger.log(Level.SEVERE, "Failed to launch CraftserveRadiation. Plausibly your server version is unsupported.", e);
-            this.setEnabled(false);
-            return;
-        }
-
```

Comment:
I would keep this here. If we can't run because of unsupported software, fail as early as possible.